corehq/apps/linked_domain/views.py

```diff
@@ -502,6 +503,7 @@ def delete_domain_link(self, in_data):
 
         if toggles.ERM_DEVELOPMENT.enabled(self.domain):
             _ = unlink_apps_in_domain(linked_domain)
+            _ = unlink_reports_in_domain(linked_domain)
```

Comment:
`_` is used so frequently as a translations alias that it's a dangerous variable name.